e2e/cypress/integration/notifications/desktop_notifications_spec.js

```diff
@@ -218,6 +218,57 @@ describe('Desktop notifications', () => {
             });
         });
     });
+
+    it('MM-T885 Channel notifications: Desktop notifications mentions only', () => {
+        cy.apiCreateUser().then(({user}) => {
+            cy.apiAddUserToTeam(testTeam.id, user.id);
+            cy.apiLogin(user);
+
+            // Visit town-square.
+            cy.visit(`/${testTeam.name}/channels/town-square`);
+            spyNotificationAs('withNotification', 'granted');
+
+            // # Ensure notifications are set up to fire a desktop notification
+            changeDesktopNotificationSettingsAs('#desktopNotificationAllActivity');
+
+            cy.apiGetChannelByName(testTeam.name, 'Off-Topic').then(({channel}) => {
+                const messageWIthNotification = `random message with mention @${user.username}`;
+
+                // # Go to Off topic
+                cy.visit(`/${testTeam.name}/channels/${channel.name}`);
+
+                // # Set channel notifications to show on mention only
+                cy.get('#channelHeaderDropdownIcon').click();
+                cy.findByText('Notification Preferences').click();
+                cy.findByText('Send desktop notifications').click();
+                cy.get('#channelNotificationMentions').click();
+                cy.get('#saveSetting').click();
+
+                // # Visit Town square
+                cy.visit(`/${testTeam.name}/channels/town-square`);
```

Comment:
Stub to notification didn't work on me since a page has been reloaded up until here. I believe it should be done here, and not the first visit to town-square.

Reply:
I need some help with this stub. It's not working for me anymore :/